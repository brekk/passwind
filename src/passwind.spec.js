import path from 'path'
// import { pipe, map } from 'ramda'
import { fork } from 'fluture'
// import {hook} from 'ripjam/test'
import { passwind } from './index'
const local = x => path.join(__dirname, x)

const fixture = {
  css: local('../fixture/example.css'),
  html: local('../fixture/example.html')
}

test('passwind', done => {
  fork(done)(result => {
    expect(result).toMatchSnapshot()
    done()
  })(passwind(fixture.html, fixture.css))
})
