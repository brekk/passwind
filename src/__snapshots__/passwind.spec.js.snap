// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passwind 1`] = `
Object {
  ".-translate-x-full": Object {
    "--tw-translate-x": "-100%",
  },
  ".absolute": Object {
    "position": "absolute",
  },
  ".bg-alertyellow": Object {
    "--tw-bg-opacity": "1",
    "backgroundColor": "rgba(255, 241, 138, var(--tw-bg-opacity))",
  },
  ".bg-black": Object {
    "--tw-bg-opacity": "1",
    "backgroundColor": "rgba(0, 0, 0, var(--tw-bg-opacity))",
  },
  ".bg-ifpink": Object {
    "--tw-bg-opacity": "1",
    "backgroundColor": "rgba(255, 194, 235, var(--tw-bg-opacity))",
  },
  ".bg-transparent": Object {
    "backgroundColor": "transparent",
  },
  ".bg-white": Object {
    "--tw-bg-opacity": "1",
    "backgroundColor": "rgba(255, 255, 255, var(--tw-bg-opacity))",
  },
  ".border": Object {
    "borderWidth": "1px",
  },
  ".border-2": Object {
    "borderWidth": "2px",
  },
  ".border-black": Object {
    "--tw-border-opacity": "1",
    "borderColor": "rgba(0, 0, 0, var(--tw-border-opacity))",
  },
  ".border-solid": Object {
    "borderStyle": "solid",
  },
  ".border-t": Object {
    "borderTopWidth": "1px",
  },
  ".fill-current": Object {
    "fill": "currentColor",
  },
  ".flex": Object {
    "display": "flex",
  },
  ".flex-1": Object {
    "flex": "1 1 0%",
  },
  ".flex-col": Object {
    "flexDirection": "column",
  },
  ".font-extended": Object {
    "fontFamily": "extended-regular, helvetica, sans-serif",
  },
  ".font-favorit": Object {
    "fontFamily": "favorit-regular, helvetica, sans-serif",
  },
  ".h-10": Object {
    "height": "2.5rem",
  },
  ".h-12": Object {
    "height": "3rem",
  },
  ".h-16": Object {
    "height": "4rem",
  },
  ".h-auto": Object {
    "height": "auto",
  },
  ".h-full": Object {
    "height": "100%",
  },
  ".h-screen": Object {
    "height": "100vh",
  },
  ".hidden": Object {
    "display": "none",
  },
  ".items-center": Object {
    "alignItems": "center",
  },
  ".items-stretch": Object {
    "alignItems": "stretch",
  },
  ".justify-between": Object {
    "justifyContent": "space-between",
  },
  ".justify-center": Object {
    "justifyContent": "center",
  },
  ".leading-loose": Object {
    "lineHeight": 2,
  },
  ".leading-relaxed": Object {
    "lineHeight": 1.625,
  },
  ".m-auto": Object {
    "margin": "auto",
  },
  ".max-h-16": Object {
    "maxHeight": "4rem",
  },
  ".mb-16": Object {
    "marginBottom": "4rem",
  },
  ".mb-2": Object {
    "marginBottom": "0.5rem",
  },
  ".mb-4": Object {
    "marginBottom": "1rem",
  },
  ".mb-8": Object {
    "marginBottom": "2rem",
  },
  ".mb-px3": Object {
    "marginBottom": "0.1875rem",
  },
  ".min-h-screen": Object {
    "minHeight": "100vh",
  },
  ".ml-2": Object {
    "marginLeft": "0.5rem",
  },
  ".ml-4": Object {
    "marginLeft": "1rem",
  },
  ".mt-1": Object {
    "marginTop": "0.25rem",
  },
  ".mt-16": Object {
    "marginTop": "4rem",
  },
  ".mt-2": Object {
    "marginTop": "0.5rem",
  },
  ".mt-3": Object {
    "marginTop": "0.75rem",
  },
  ".mt-8": Object {
    "marginTop": "2rem",
  },
  ".mx-1": Object {
    "marginLeft": "0.25rem",
    "marginRight": "0.25rem",
  },
  ".my-10": Object {
    "marginBottom": "2.5rem",
    "marginTop": "2.5rem",
  },
  ".offset-box": Object {
    "border": "2px solid #000",
    "position": "relative",
    "transformStyle": "preserve-3d",
  },
  ".p-2": Object {
    "padding": "0.5rem",
  },
  ".p-3": Object {
    "padding": "0.75rem",
  },
  ".p-4": Object {
    "padding": "1rem",
  },
  ".px-0": Object {
    "paddingLeft": "0",
    "paddingRight": "0",
  },
  ".px-2": Object {
    "paddingLeft": "0.5rem",
    "paddingRight": "0.5rem",
  },
  ".px-3": Object {
    "paddingLeft": "0.75rem",
    "paddingRight": "0.75rem",
  },
  ".px-5": Object {
    "paddingLeft": "1.25rem",
    "paddingRight": "1.25rem",
  },
  ".px-6": Object {
    "paddingLeft": "1.5rem",
    "paddingRight": "1.5rem",
  },
  ".py-14": Object {
    "paddingBottom": "3.5rem",
    "paddingTop": "3.5rem",
  },
  ".py-3": Object {
    "paddingBottom": "0.75rem",
    "paddingTop": "0.75rem",
  },
  ".py-7": Object {
    "paddingBottom": "1.75rem",
    "paddingTop": "1.75rem",
  },
  ".py-8": Object {
    "paddingBottom": "2rem",
    "paddingTop": "2rem",
  },
  ".relative": Object {
    "position": "relative",
  },
  ".rounded-full": Object {
    "borderRadius": "9999px",
  },
  ".rounded-plus": Object {
    "borderRadius": "0.3125rem",
  },
  ".text-4xl": Object {
    "fontSize": "2.25rem",
    "lineHeight": "2.5rem",
  },
  ".text-black": Object {
    "--tw-text-opacity": "1",
    "color": "rgba(0, 0, 0, var(--tw-text-opacity))",
  },
  ".text-center": Object {
    "textAlign": "center",
  },
  ".text-gray-500": Object {
    "--tw-text-opacity": "1",
    "color": "rgba(107, 114, 128, var(--tw-text-opacity))",
  },
  ".text-ifblue": Object {
    "--tw-text-opacity": "1",
    "color": "rgba(29, 0, 112, var(--tw-text-opacity))",
  },
  ".text-iflightblue": Object {
    "--tw-text-opacity": "1",
    "color": "rgba(3, 60, 224, var(--tw-text-opacity))",
  },
  ".text-lg": Object {
    "fontSize": "1.125rem",
  },
  ".text-sm": Object {
    "fontSize": "0.875rem",
    "lineHeight": "1.25rem",
  },
  ".text-white": Object {
    "--tw-text-opacity": "1",
    "color": "rgba(255, 255, 255, var(--tw-text-opacity))",
  },
  ".text-xl": Object {
    "fontSize": "1.25rem",
  },
  ".text-xs": Object {
    "fontSize": "0.75rem",
    "lineHeight": "1rem",
  },
  ".transform-gpu": Object {
    "--tw-rotate": "0",
    "--tw-scale-x": "1",
    "--tw-scale-y": "1",
    "--tw-skew-x": "0",
    "--tw-skew-y": "0",
    "--tw-translate-x": "0",
    "--tw-translate-y": "0",
    "transform": "translate3d(
      var(--tw-translate-x),
      var(--tw-translate-y),
      0
    )
    rotate(var(--tw-rotate)) skewX(var(--tw-skew-x))
    skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x))
    scaleY(var(--tw-scale-y))",
  },
  ".transition": Object {
    "transitionDuration": "0.15s",
    "transitionProperty": Array [
      "background-color, border-color, color, fill,
    stroke, opacity, box-shadow, transform, filter,
    -webkit-backdrop-filter",
      "background-color, border-color, color, fill,
    stroke, opacity, box-shadow, transform, filter, backdrop-filter",
      "background-color, border-color, color, fill,
    stroke, opacity, box-shadow, transform, filter, backdrop-filter,
    -webkit-backdrop-filter",
    ],
    "transitionTimingFunction": "cubic-bezier(0.4, 0, 0.2, 1)",
  },
  ".transition-all": Object {
    "transitionDuration": "0.15s",
    "transitionProperty": "all",
    "transitionTimingFunction": "cubic-bezier(0.4, 0, 0.2, 1)",
  },
  ".w-40": Object {
    "width": "10rem",
  },
  ".w-52": Object {
    "width": "13rem",
  },
  ".w-60": Object {
    "width": "15rem",
  },
  ".w-full": Object {
    "width": "100%",
  },
  ".w-screen": Object {
    "width": "100vw",
  },
  ".whitespace-nowrap": Object {
    "whiteSpace": "nowrap",
  },
  ".z-10": Object {
    "zIndex": 10,
  },
  ".z-20": Object {
    "zIndex": 20,
  },
}
`;
